README.md

```diff
@@ -8,20 +8,19 @@
 [![](https://img.shields.io/pypi/l/sentinelhub.svg)](https://github.com/sentinel-hub/sentinelhub-py/blob/master/LICENSE.md)
 [![Code coverage](https://codecov.io/gh/sentinel-hub/sentinelhub-py/branch/master/graph/badge.svg)](https://codecov.io/gh/sentinel-hub/sentinelhub-py)
 
-# Description
+## Introduction
 
-The **sentinelhub** Python package allows users to make OGC (WMS and WCS)
-web requests to download and process satellite images within your Python
-scripts. It supports Sentinel-2 L1C and L2A, Sentinel-1, Landsat 8, MODIS and DEM data source.
+The **sentinelhub** Python package is the official Python interface for [Sentinel Hub services](https://www.sentinel-hub.com/). It supports most of the services described in the [Sentinel Hub documentation](https://docs.sentinel-hub.com/api/latest/) and any type of [satellite data collections](https://docs.sentinel-hub.com/api/latest/data/), including Sentinel, Landsat, MODIS, DEM, and custom collections produced by users.
 
-The package also supports obtaining data from Amazon Web Service. It can either provide data from public bucket with
-Sentinel-2 L1C imagery or requester pays bucket with Sentinel-2 L2A imagery. If specified the downloaded data can be
-stored in ESA .SAFE format (all types of .SAFE format are supported).
+The package is also a collection of basic tools and utilities for working with geospatial and satellite data. It builds on top of well known packages such as `numpy`, `shapely`, `pyproj`, etc. On the other hand, it serves as a core dependency of [`eo-learn`](https://github.com/sentinel-hub/eo-learn) Python package for creating geospatial data-processing workflows.
 
-# Installation
+The main package resources are [GitHub repository](https://github.com/sentinel-hub/sentinelhub-py), [documentation page](https://sentinelhub-py.readthedocs.io/en/latest/) and [Sentinel Hub forum](https://forum.sentinel-hub.com/).
 
-The package requires a Python version >= 3.6 and installed C/C++ compiler. The package is available on
-the PyPI package manager and can be installed with
+
+## Installation
+
+The package requires a Python version >= 3.6 and installed C/C++ compiler. The package is available at
```

Comment:
and installed C/C++ compiler
-> and **an** installed C/C++ compiler

Reply:
fixed